source/collections/Vector2D.ooc

```diff
@@ -35,19 +36,74 @@ Vector2D: class <T> {
 	free: override func {
 		gc_free(this _backend)
 	}
-	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
+	_elementPosition: func (row, column: Int, columnCount := this columnCount) -> Int {
 		result := columnCount * row + column
```

Comment:
Maybe, sometimes, rock returns the last created variable...? Anyway, yeah, the `result` variable is unnecessary.